Dockerfile

```diff
@@ -73,8 +77,6 @@ EXPOSE 8081
 USER nexus
 WORKDIR ${NEXUS_HOME}
 
-ENV JAVA_MAX_MEM=1200m \
-  JAVA_MIN_MEM=1200m \
-  EXTRA_JAVA_OPTS=""
+ENV INSTALL4J_ADD_VM_PARAMS="-Xms1200m -Xmx1200m"
```

Comment:
if the user forgets to include -Xms -Xmx args in this environment variable then there will be no max or min heap set at all because of line 67. Is that what we want?